controller/resource.go

```diff
@@ -173,7 +179,35 @@ func (r *ResourceRepo) Remove(id string) error {
 		tx.Rollback()
 		return err
 	}
-	return tx.Commit()
+	if err := tx.Commit(); err != nil {
+		return err
+	}
+	r.createResourceDeletionEvent(providerID, id)
+	return nil
+}
+
+func (r *ResourceRepo) createResourceEvent(resource *ct.Resource) {
+	logErr := func(err error) {
+		log.Printf("Error creating resource event: %s\n", err)
+	}
+	data, err := json.Marshal(resource)
+	if err != nil {
+		logErr(err)
+		return
+	}
+	objectID := strings.Join([]string{resource.ProviderID, resource.ID}, ":")
+	query := "INSERT INTO events (object_id, object_type, data) VALUES ($1, $2, $3)"
```

Comment:
Resources can be associated with many apps, so does it make sense to insert this event for all the associated apps?